src/main/java/John.java

```diff
@@ -0,0 +1,161 @@
+import commands.Deadline;
+import commands.Event;
+import commands.Task;
+import commands.Todo;
+
+
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.List;
+
+public class John {
+    //Load in the data and import it into the task list
+    static List<Task> tasks;
+    static {
+        try {
+            tasks = DataManager.readData("src/main/java/data.txt");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void main(String[] args) {
+        System.out.println("ヽ(•‿•)ノ");
+         System.out.println("-----------------------------------");
+         System.out.println("Heya, Im ya Chatbot, call me John!\nWhat can I do for you today?");
+         // Taking input from the user
+         getUserInput();
+         System.out.println("\n---------------------------------");
+         System.out.println("Hope to see you soon Bruda.");
+    }
+
+    // Method to take input and return it
+    public static void getUserInput() {
+        // Creating a Scanner object for input
+        Scanner scanner = new Scanner(System.in);
+
+        boolean isFinished = false;
+
+        while (!isFinished) {
+            // Prompting the user for input
+            System.out.print("Enter command: ");
+            // Reading the input
+            String input = scanner.nextLine();
+
+            if (input.equals("Bye") || input.equals("bye")){
+                // Closing the Scanner to avoid resource leaks
+                scanner.close();
+                isFinished = true;
+            } else if (input.equals("List") || input.equals("list")){
+                for (int i=0; i<tasks.size(); i++){
```

Comment:
```suggestion
                for (int i=0; i<tasks.size(); i++) {
```

Take note of the form of the loop.